clusterloader2/pkg/prometheus/experimental.go

```diff
@@ -86,7 +86,7 @@ func (pc *Controller) tryRetrievePrometheusDiskMetadata() (bool, error) {
 		}
 		klog.V(2).Infof("Found Prometheus' PV with name: %s", pv.Name)
 		pdName = pv.Spec.GCEPersistentDisk.PDName
-		zone = pv.ObjectMeta.Labels["failure-domain.beta.kubernetes.io/zone"]
+		zone = pv.ObjectMeta.Labels["topology.kubernetes.io/zone"]
```

Comment:
For some reason prometheus' disk is still using failure-domain.beta.kubernetes.io.zone: https://github.com/kubernetes/kubernetes/issues/96509#issuecomment-725978098